jdaviz/cli.py

```diff
@@ -115,6 +120,31 @@ def _main():
     parser.add_argument('--version', action='version', version=f'%(prog)s {__version__}')
     args = parser.parse_args()
 
-    main(args.filename, layout=args.layout, instrument=args.instrument, browser=args.browser,
+    if config is None:
+        layout = args.layout
+    else:
+        layout = config
+
+    main(filename=args.filename, layout=layout, instrument=args.instrument, browser=args.browser,
          theme=args.theme, verbosity=args.verbosity, history_verbosity=args.history_verbosity,
          hotreload=args.hotreload)
+
+
+def _specviz():
```

Comment:
Do we really need this?

Reply:
I assume you're referring to the config launchers here? We don't technically need them, but it makes launching the configs much easier from the command line. Until the freeform is more fully fleshed-out, our users are *really* installing `jdaviz` to get access to the configs.

Also, to peel back the curtain a little bit on the installer work, these entry_points create an executable in the virtualenv (AKA the thing you call when you run `$ jdaviz`. I learned you can create desktop shortcuts to these executables so you can "double click" them, and they'll launch. Having separate `specviz.exe`, `cubeviz.exe`, `imviz.exe` executables allows me to simply create a shortcut to the relevant executable. If we don't have these config launchers, I'll need to create shortcuts to the existing `jdaviz.exe` and pass the config as an argument in each shortcut. Doable, but more complex

Reply:
The other options from a desktop shortcut would be to launch a splash page where you then choose which config to load.  I have slight concerns about eventual clashes as we make more configs, but think its worth it for the convenience (and maybe we could eventually have most customized installation where the user picks which to setup/install).

Reply:
I am also scared of having so many executables from one install. Also please make sure they go away cleanly on uninstall.

Reply:
Sure, but if I uninstall from that environment, I also expect them to die in that environment without having to delete the whole environment to get rid of them. If they are registered properly with Python during install, should just happen but best to double check.

Reply:
All gone!

```
(envtest) PS E:\STScI\gitRepos\jdaviz> pip uninstall jdaviz
Found existing installation: jdaviz 3.2.dev175+ge3b9d944.d20221205
Uninstalling jdaviz-3.2.dev175+ge3b9d944.d20221205:
  Would remove:
    e:\stsci\gitrepos\jdaviz\envtest\lib\site-packages\jdaviz-3.2.dev175+ge3b9d944.d20221205.dist-info\*
    e:\stsci\gitrepos\jdaviz\envtest\lib\site-packages\jdaviz\*
    e:\stsci\gitrepos\jdaviz\envtest\scripts\cubeviz.exe
    e:\stsci\gitrepos\jdaviz\envtest\scripts\imviz.exe
    e:\stsci\gitrepos\jdaviz\envtest\scripts\jdaviz.exe
    e:\stsci\gitrepos\jdaviz\envtest\scripts\mosviz.exe
    e:\stsci\gitrepos\jdaviz\envtest\scripts\specviz.exe
    e:\stsci\gitrepos\jdaviz\envtest\scripts\specviz2d.exe
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\nbconvert\templates\jdaviz-default\ansi.js
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\nbconvert\templates\jdaviz-default\app.html
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\nbconvert\templates\jdaviz-default\conf.json
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\nbconvert\templates\jdaviz-default\index.html.j2
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\nbconvert\templates\jdaviz-default\util.js
    e:\stsci\gitrepos\jdaviz\envtest\share\jupyter\voila\templates\jdaviz-default\index.html.j2
Proceed (Y/n)? y
  Successfully uninstalled jdaviz-3.2.dev175+ge3b9d944.d20221205
(envtest) PS E:\STScI\gitRepos\jdaviz> ls .\envtest\Scripts\*viz.exe
(envtest) PS E:\STScI\gitRepos\jdaviz>
```

Reply:
As per the plan in https://github.com/spacetelescope/jdaviz/pull/1960#issuecomment-1444405656, I reverted this change, so this point is now not applicable